smriprep/cli/run.py

```diff
@@ -402,12 +402,15 @@ def build_workflow(opts, retval):
                                                  packagename="smriprep")
         return retval
 
+    output_spaces = opts.output_spaces
+    output_spaces.checkpoint()
```

Comment:
I needed to checkpoint to be able to read the references' full specs down the line.